bundle/edu.gemini.auxfile.workflow/src/main/java/edu/gemini/auxfile/client/AuxFileClient.java

```diff
@@ -21,15 +21,17 @@
  */
 public final class AuxFileClient implements AuxFileSystem {
 
-    private final int chunkSize;
+    private static final int FileChunkSize = 32 * 1024;
+
     private final AuxFileServer server;
 
-    public AuxFileClient(KeyChain kc, int chunkSize, String host, int port) {
-        this.chunkSize = chunkSize;
-        this.server    = TrpcClient$.MODULE$.apply(host, port).withKeyChain(kc).proxy(AuxFileServer.class);
+    public AuxFileClient(String host, int port) {
+        this.server    = TrpcClient$.MODULE$.apply(host, port).withoutKeys().proxy(AuxFileServer.class);
```

Comment:
To my surprise, unless I'm missing something, we don't seem to be doing any permissions checking for auxfile operations on the server side.  That seems poor, though obviously not the fault of this PR. We _should_ require a valid key to manipulate auxfiles though, at which point this constructor won't be very useful.

That raises the question of how this would work if the auxfile server were doing permission checking.  The ODB itself (or whatever process the ITC service is running in) would need a staff key.